root/artist/index.tt

```diff
@@ -35,7 +35,9 @@ END %]
 
     [%- IF release_groups.size -%]
       <form action="[% c.uri_for_action('/release_group/merge_queue') %]" method="post">
-        [% INCLUDE 'components/release_groups-list.tt' release_groups=release_groups checkboxes='add-to-merge' WRAPPER 'components/with-pager.tt' %]
+        [% # TODO: MBS-10155 %]
+        [% React.embed(c, 'components/ReleaseGroupsList', { releaseGroups => release_groups, checkboxes => 'add-to-merge', showRatings => 1, groupByType => 1 })
```

Comment:
I suggest to comment in line (so as not to output an extra line) and to detail when to check it:
```suggestion
        [% React.embed(c, 'components/ReleaseGroupsList', { releaseGroups => release_groups, checkboxes => 'add-to-merge', showRatings => 1, groupByType => 1 }) # TODO: On conversion to React, please check MBS-10155.
```